Functions/New-MockObject.Tests.ps1

```diff
@@ -0,0 +1,10 @@
+describe 'New-MockObject' {
+
+    . .\New-MockObject.ps1
```

Comment:
There's no need for this.  Pester files are already loaded when we're running our own tests, and for some reason this line is failing anyway.